### PR TITLE
ENG-1593 disable body scroll lock for mobile devices

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/app-tour/AppTour.js
Comment (raxkaynan) [inline src/ui/app-tour/AppTour.js:409]: I think `scrollLockMobile` is quite confusing since it means the opposite of what the condition (screen width > 1000) means.

**Review by raxkaynan — CHANGES_REQUESTED** [review]

**Review by raxkaynan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/973#pullrequestreview-560582698" title="Approved">raxkaynan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [raxkaynan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
